nbclient/util.py

```diff
@@ -81,13 +81,6 @@ async def ensure_async(obj: Union[Awaitable, Any]) -> Any:
     and await it if it was not already awaited.
```

Comment:
The docstring has to be changed too, since we always await if the object is a coroutine.